R/visualization.R

```diff
@@ -3604,7 +3604,7 @@ ExIPlot <- function(
     FUN = function(x) {
       return(SingleExIPlot(
         type = type,
-        data = data[, x, drop = FALSE],
+        data = data[, features, drop = FALSE],
```

Comment:
Why change this to `features`? I believe `SingleExIPlot` expects a single column data.frame.

Reply:
That's unintended, I meant to stash and discard that. I'll remove

Reply:
Done, should be up to date with devel also